.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Before submitting a pull request, please make sure the following is done:
+
+- [ ] Fork the repository and create your branch from master.
+- [ ] If you've added code that should be tested, add tests!
+- [ ] If you've changed APIs, update the documentation in [README](/) and [api/README](api/README.md)
+- [ ] If you've made a notable change (major enhancement, new feature) be sure to add it to the unreleased section in the [CHANGELOG](CHANGELOG.md)
+- [ ] Ensure the test suite passes (npm test).
+- [ ] Format your code with prettier (npm run prettier).
```

Comment:
Was just thinking the same thing. [lint-staged](https://github.com/okonet/lint-staged) or another preference?